File: iPad/VerBank-CSTSv3-IOSClientApi/ClientStation/api/info/InfoOperator_TRADESERV1016.m
//
//  InfoOperator_TRADESERV1016.m
//  VerBank-IOSClientStation
//
//  Created by Allone on 15/8/31.
//  Copyright (c) 2015年 zhanglei. All rights reserved.
//

#import "InfoOperator_TRADESERV1016.h"

#import "Info_TRADESERV1016.h"
#import "APIDocCaptain.h"

@implementation InfoOperator_TRADESERV1016

- (void)onInfo:(InfoFather *)info {
    Info_TRADESERV1016 *tinfo = (Info_TRADESERV1016 *)info;
    [APIDocCaptain setClosedInstruments:[tinfo getClosedInstruments]];
}

@end

File: WeChatService/EX02-CSTSv3-ClientAPI/src/jedi/ex01/client/station/api/info/InfoOperator_TRADESERV1016.java
package jedi.ex01.client.station.api.info;

import jedi.ex01.CSTS3.comm.info.InfoFather;
import jedi.ex01.CSTS3.comm.info.Info_TRADESERV1016;
import jedi.ex01.client.station.api.doc.util.DocUtil;

public class InfoOperator_TRADESERV1016 extends InfoOperator {

	@Override
	public void onInfo(InfoFather _info) {
		 Info_TRADESERV1016 info = (Info_TRADESERV1016) _info;
		 DocUtil.setClosedInstruments(info.getClosedInstruments());
//		 APIDocCaptain.setTimedoutQuote(info.getTimedoutInstruments());
	}
}
